modules/foreman_bookmark.py

```diff
@@ -32,8 +32,8 @@
 version_added: "2.4"
 author:
   - "Bernhard Hopfenmueller (@Fobhep) ATIX AG"
+  - "Christoffer Reijer (@ephracis) Basalt AB"
 requirements:
-  - "nailgun >= 0.29.0"
```

Comment:
:tada: every time i see it.

Reply:
ACK, but probably this needs a `apypie` here now?

Reply:
Done.